scripts/run_tests.sh

```diff
@@ -6,10 +6,11 @@ set -e
 (cd /app/relation_engine_server/test/spec_release && \
   tar czvf spec.tar.gz sample_spec_release)
 
-black .
-flake8 --max-complexity 20 /app
-mypy --ignore-missing-imports /app
-bandit -r /app
+# TODO: why reformatting code?
```

Comment:
remove these comments

Reply:
unfortunately, reverting the prior to commits also reverted some changes like this.